EIPS/eip-2309.md

```diff
@@ -101,25 +89,37 @@ The `ConsecutiveTransfer` event can be used for a single token as well as many t
 `emit ConsecutiveTransfer(1, 100000, from, address(0));`
 
 ---
```

Comment:
If you decide to use a sub-heading for the examples, then consider removing this horizontal rule
```suggestion
```

Reply:
Thanks. Updated.